### PR TITLE
fixed contributions and repositories bug in sidebar of resume

### PR DESCRIPTION
**Pull request by atharvarekhawar**
## Related Issue
None

## Description
There was a rendering bug in show contributions which is no more and the limit of repositories is increased to 6.

## Type of PR

- [x] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist:
- [X ] I have performed a self-review of my code
- [X ] I have read and followed the Contribution Guidelines.
- [X ] I have tested the changes thoroughly before submitting this pull request.
- [X ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X ] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
Please add appropriate labels while merging, Thank you 😊.

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
Thank you for submitting your pull request! 🙌 We'll review it as soon as possible. In the meantime, please ensure that your changes align with our [CONTRIBUTING.md](https://github.com/ashutosh-rath02/git-re/blob/main/CONTRIBUTING.md). If there are any specific instructions or feedback regarding your PR, we'll provide them here. Thanks again for your contribution! 😊

**Comment by ashutosh-rath02** [comment]
@atharvarekhawar Can you share a video with the changes, it would be helpful?

**Comment by atharvarekhawar** [comment]
> @atharvarekhawar Can you share a video with the changes, it would be helpful?


https://github.com/ashutosh-rath02/git-re/assets/113119063/ab20ecbf-f869-4fc2-a9a7-cfd7365795c4

@ashutosh-rath02 now there is no problem while changing show contributions count and contributions are rendered without any glitch , show repositories have been limited to 6 and if user inserts number larger than 6 , still  it shows 6 repos.

File: utils/resumeUtils.ts [hunk 1]
```diff
@@ -210,6 +210,9 @@ export const fetchPopularRepos = async (username: string): Promise<Repo[]> => {
     );
     const repos = response.data;
 
+    console.log("popular",repos);
+    
+
```
Comment (ashutosh-rath02) [inline utils/resumeUtils.ts:215]: remove this

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

**Comment by ashutosh-rath02** [comment]
Can you explain me the bug again once coz I see only changes, where you changed limit to 5

**Comment by atharvarekhawar** [comment]
> Can you explain me the bug again once coz I see only changes, where you changed limit to 6

The glitch of the contributions was not present when I pulled latest changes and started the application on my local machine , so I just changed the limit to 6.

@ashutosh-rath02

File: utils/resumeUtils.ts (hunk 1 above)
Comment (atharvarekhawar) [reply utils/resumeUtils.ts]: done with the changes @ashutosh-rath02

**Comment by atharvarekhawar** [comment]
can you merge the code @ashutosh-rath02

**Comment by ashutosh-rath02** [comment]
This doesn't fix the problem as provided in the parent issue.

**Comment by atharvarekhawar** [comment]
> This doesn't fix the problem as provided in the parent issue.

there is no bug at the current moment in the website which is similar to the parent issue

**Review by ashutosh-rath02 — COMMENTED** [review]
LGTM

**Pull request merged**